client/app/constants/SearchPageConstants.js

```diff
@@ -0,0 +1,6 @@
+// Search page
+const CURRENT_PAGE_UPDATE = 'SEARCH::CURRENT_PAGE_UPDATE';
+
+export {
+  CURRENT_PAGE_UPDATE,
+};
```

Comment:
I'd want to keep onboarding away from this data cycle => searchpage and everything inside of it could be added into this file (including topbar search inputs). So, I agree, but don't want to do any changes to current files.